libiocage/lib/JailConfig.py

```diff
@@ -199,9 +197,8 @@ def _get_clonejail(self):
     def _default_clonejail(self):
         return True
 
-    def _set_clonejail(self, value, **kwargs):
-        self.data["clonejail"] = "on" if (value is True) or (
-                                         value == "on") else "off"
+    def _set_clonejail(self, val, **kwargs):
+        self.data["clonejail"] = libiocage.lib.helpers.get_str_bool(val, "jailconfig")
```

Comment:
🙄 In addition to previous comment, I'm in favor of this style:
```
        self.data["clonejail"] = libiocage.lib.helpers.get_str_bool(
            val,
            "jailconfig"
        )
```

Please let us not randomize wrapping. And @igalic I got your point that an 80 character limit is randomly choosen. The two lines you deleted here before looked super ugly. That destroys all benefit 80 characters width can have, because my eyes refuse to read it.. :(